cluster-autoscaler/main.go

```diff
@@ -285,3 +286,44 @@ func main() {
 		})
 	}
 }
+
+func DefaultLeaderElectionConfiguration() componentconfig.LeaderElectionConfiguration {
```

Comment:
Ok, but let's clean this up next week (preferably by moving it to new file under /config).